tool/xtt.c

```diff
@@ -51,6 +52,9 @@ int main(int argc, char **argv)
             out = xtt_generate_server_certificate(params.rootcert, params.rootpriv, params.serverpriv,
                                                 params.serverpub, params.server_id, params.time, params.servercert);
             break;
+        case action_infocert:
+            info_cert(params.cert, params.servercert, params.rootcert);
```

Comment:
Error handling is inconsistent.  The other actions are handled in the following code block based on the `out` value. For `action_infocert`, it is handled in `info_cert()` itself.

Both should be done in the same way.

(It's fine to keep this commit as is, if you follow it with a later commit that reworks the error handling to be consistent.)